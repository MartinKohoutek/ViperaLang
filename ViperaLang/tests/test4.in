def greet():
    if True:
        print("Inside if")
    else:
        print("Inside else")
    print("Outside if-else")