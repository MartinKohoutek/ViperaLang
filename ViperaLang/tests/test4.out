IDENTIFIER def null
IDENTIFIER greet null
LPAREN ( null
RPAREN ) null
COLON : null
NEWLINE 
 null
INDENT      null
IDENTIFIER if null
IDENTIFIER True null
COLON : null
NEWLINE 
 null
INDENT          null
IDENTIFIER print null
LPAREN ( null
STRING "Inside if" Inside if
RPAREN ) null
NEWLINE 
 null
DEDENT      null
IDENTIFIER else null
COLON : null
NEWLINE 
 null
INDENT          null
IDENTIFIER print null
LPAREN ( null
STRING "Inside else" Inside else
RPAREN ) null
NEWLINE 
 null
DEDENT      null
IDENTIFIER print null
LPAREN ( null
STRING "Outside if-else" Outside if-else
RPAREN ) null
DEDENT ) null
EOF  null